packages/wix-ui-tpa/src/components/DatePicker/DatePicker.st.css

```
@@ -23,15 +23,14 @@
 :vars {
     calendarWidth: 100%;
     calendarHeight: 100%;
-    sidePadding: 20px;
+    sidePadding: 21px;
 
```

Maybe we want to set `14` as a variable?
```suggestion
    minFontSize: 14;
```

I see we use it in 3 calculations (lines 151, 213, 241)